content/SCALE/SCALETutorials/Credentials/ManageLocalUsersSCALE.md

```diff
@@ -30,13 +31,12 @@ Using [Active Directory]({{< relref "/content/SCALE/SCALEUIReference/Credentials
 
 To see user accounts, go to **Credentials > Local Users**.
 
-![LocalUsersSCALE](/images/SCALE/22.12/LocalUsersSCALE.png "List of Local User Accounts") 
+![AllUsersScreenSCALE](/images/SCALE/22.12/AllUsersScreenSCALE.png "Local User non-Built-in Accounts") 
 
 TrueNAS hides all built-in users (except root) by default. Click the toggle **Show Built-In Users** to see all built-in users.
 
 ## Creating an Admin User Account
-SCALE has implemented rootless log in. All systems should create and begin using an admin user as a replacement for the root user. 
-A system warning alert displays until you create the admin user. 
+SCALE has implemented rootless log in. All systems should create and begin using an admin user as a replacement for the root user.  
```

Comment:
```suggestion
SCALE has implemented rootless login. All systems should create and use an admin user to replace the root user.  
```

Reply:
Claire, I'm not sure how your article got into my PR. Would you review this open comment and either accept or change the suggestion made by Tony.